generators/server/templates/dotnetcore/src/Project/Project.csproj.ejs

```diff
@@ -37,13 +37,38 @@
         <PackageReference Include="AutoMapper.Extensions.Microsoft.DependencyInjection" Version="6.0.0" />
         <PackageReference Include="Hellang.Middleware.ProblemDetails" Version="2.2.1" />
         <PackageReference Include="Microsoft.AspNetCore.Authentication.JwtBearer" Version="3.1.1" />
-        <PackageReference Include="Microsoft.AspNetCore.Identity.EntityFrameworkCore" Version="3.1.1" />
         <PackageReference Include="Microsoft.AspNetCore.Mvc" Version="2.2.0" />
         <PackageReference Include="Microsoft.AspNetCore.Mvc.NewtonsoftJson" Version="3.1.1" />
         <PackageReference Include="Microsoft.AspNetCore.Mvc.Razor.RuntimeCompilation" Version="3.1.0" Condition="'$(Configuration)' == 'Debug'" />
         <PackageReference Include="Microsoft.AspNetCore.Mvc.Analyzers" Version="2.2.0" />
-        <PackageReference Include="Microsoft.EntityFrameworkCore" Version="3.1.1" />
+        <%_ switch(databaseType) {
+            case 'sqllite': _%>
+        <PackageReference Include="Microsoft.AspNetCore.Identity.EntityFrameworkCore" Version="3.1.1" />
```

Comment:
We may be able to factorize this dependency (every DB use it), but don't feel obligated. How it behaves with Mongo `Microsoft.AspNetCore.Identity.EntityFrameworkCore` support it too?

Reply:
Mongo is big work in progress for now, I'm trying to use https://github.com/tugberkugurlu/AspNetCore.Identity.MongoDB for identity, we will create another PR if it works out

Reply:
👍 for another PR for mongo